# This expresses a special action requests.

# ENUM(SpecialActionID: playDead, standUpBackNao, standUpFrontNao, sitDown, sitDownKeeper, goUp, keeperJumpLeftSign);

uint8 specialAction  #; /**< The special action selected. */
bool mirror          #; /**< Mirror left and right. */


# FALTA getSpecialActionFromName
